mypyc/irbuild/format_str_tokenizer.py

```diff
@@ -186,3 +191,51 @@ def join_formatted_strings(builder: IRBuilder, literals: Optional[List[str]],
 
     result_list[0] = Integer(len(result_list) - 1, c_pyssize_t_rprimitive)
     return builder.call_c(str_build_op, result_list, line)
+
+
+def convert_expr_to_bytes(builder: IRBuilder, format_ops: List[FormatOp],
+                          exprs: List[Expression], line: int) -> Optional[List[Value]]:
+    """Convert expressions into bytes literals with the guidance
```

Comment:
Nit: This doesn't convert them into literals but bytes objects?

Document None return value.